napari/components/experimental/monitor/_api.py

```diff
@@ -81,12 +81,12 @@ class MonitorApi:
     # BaseManager.register() is a bit weird. Not sure now to best deal with
     # it. Most ways I tried led to pickling errors, because this class is being run
     # in the shared memory server process? Feel free to find a better approach.
-    _napari_data_dict = {}
-    _napari_messages_queue = Queue()
-    _napari_shutdown_event = Event()
+    _napari_data_dict: ClassVar[Dict] = {}
+    _napari_messages_queue: ClassVar[Queue] = Queue()
+    _napari_shutdown_event: ClassVar[Event] = Event()
 
-    _client_data_dict = {}
-    _client_messages_queue = Queue()
+    _client_data_dict: ClassVar[Dict] = {}
+    _client_messages_queue: ClassVar[Queue] = Queue()
```

Comment:
I'm not sure here. The ruff is happy, but further investigation shows that `_napari_data_dict` was passed as `callable` named argument. Is anyone feel good with this part of the codebase to validate it?